docs/source/plugin/advanced.rst

```diff
@@ -0,0 +1,67 @@
+.. _plugin-advanced:
+
+======================
+Advanced Tips & Tricks
+======================
+
+Now that you know the basics about plugins, you may have more
+questions, or there is a specific need you want to fulfill but can't quite
+grasp how to implement it. After all, this documentation alone can't cover
+every possible case!
```

Comment:
Prime example of something I couldn't review the other day on mobile: wrapping. Phone screen isn't wide enough (and I don't think GitHub's app shows indentation properly anyway). In this case I also reworded it. Beware that re-wrapping the original will cause "case!" to become a widow.

```suggestion
Now that you know the basics about plugins, you may have more questions, or
you might have a specific need but can't quite grasp how to implement it.
After all, this documentation alone can't cover every possible case!
```